lib/cmis/exceptions.rb

```diff
@@ -1,18 +1,19 @@
 module CMIS
   module Exceptions
```

Comment:
Get rid of the exceptions module, it just adds complexity.

Reply:
I'd rather not, as this will surely break backwards compatibility with certain clients of this lib that rely on rescuing these exceptions.

Also, by removing the `Exceptions` namespace, assuming we still want to enforce a common parent class for all CMIS service related exceptions/errors, we would have to write `CMIS::StorageException` iso `CMIS::Exceptions::Storage`, which to me, is the same thing.

If you feel that strongly about removing the `Exceptions` namespace a compromise would be to introduce a new exception parent for each exception that sits between `ServiceError` and the exception.

For example

```ruby
module CMIS
  ServiceError = Class.new(StandardError)
  StorageException = Class.new(ServiceError)
  module Exceptions
    Storage = Class.new(StorageException)
    ...

```